Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "onig"
-version = "4.0.0"
+version = "4.0.1"
```

Comment:
Does this need to be rebased? This version bump has already taken place in `master`.

Reply:
I've removed this commit. I'll leave version bumps to you.